doc/nimgrep.rst

```diff
@@ -2,17 +2,20 @@
   nimgrep User's manual
 =========================
 
+.. default-role:: literal
+
 :Author: Andreas Rumpf
-:Version: 0.9
+:Version: 1.6.0
```

Comment:
that version is specific to nimgrep; maybe bump it to 0.10 instead

Reply:
@timotheecour nimgrep is 1.6.0 though

Reply:
https://github.com/nim-lang/Nim/blob/devel/tools/nimgrep.nim#L14

Reply:
ok (a bit confusing since matches next nim version but i guess that's a coincidence)
btw @a-mr is there a way to insert comments to rst such that they won't render in rendered html? that would be useful for situations like this (and many others), eg:
```
.. hidden_comment::
  the version is independent from nim version
```

Reply:
yes, equal versions is a pure coincidence.